js_modules/dagit/packages/core/src/assets/AssetDefinedInMultipleReposNotice.tsx

```diff
@@ -11,16 +11,21 @@ export const AssetDefinedInMultipleReposNotice: React.FC<{
   assetId: string;
   loadedFromRepo: RepoAddress;
 }> = ({assetId, loadedFromRepo}) => {
+  function get_asset_key_from_id(id: string) {
```

Comment:
do we need to do this?

My preference is just have id be some unique string and not try to parse it (it's used internally for caching in the apollo cache).  Instead, we just add `assetKey` the asset node field and use that instead.

Reply:
That makes a lot of sense, I'll update this to use asset key instead 👍